build/x86_64_centos6/Dockerfile

```diff
@@ -11,16 +11,17 @@ RUN yum update -y \
         git \
         glibc-devel \
         patch \
-        procps \
         redhat-lsb-core \
         rpm-build \
         rpm-sign \
+        ruby \
```

Comment:
I actually suspect this is redundant with "rvm requirements", since it pulls in the packages Ruby needs to run, but it's probably harmless. This should fix the immediate issue with the container builds, and I'll send another cleanup CL if necessary.

Reply:
Sounds good.